packages/app/src/components/Settings/GrantCycleNomsResults.js

```diff
@@ -11,6 +11,12 @@ const GrantCycleNomsResults = ({ results: grantCycle, onClick }) => {
     return `${lastName}-${state}`;
   };
 
+  const redLinkHelper = (nomination) => {
```

Comment:
Javascript doesn't enforce this but it's generally a good idea to make sure your function always returns some value. This won't return anything if the if condition is false. Guessing you probably want to return an empty string instead.